planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/path_safety_checker/safety_check.cpp

```diff
@@ -722,4 +728,95 @@ void updateCollisionCheckDebugMap(
   debug_map.insert(object_debug);
 }
 
+double calcObstacleMinLength(const Shape & shape)
+{
+  if (shape.type == Shape::BOUNDING_BOX) {
+    return std::min(shape.dimensions.x / 2.0, shape.dimensions.y / 2.0);
+  } else if (shape.type == Shape::CYLINDER) {
+    return shape.dimensions.x / 2.0;
+  } else if (shape.type == Shape::POLYGON) {
+    double min_length_to_point = std::numeric_limits<double>::max();
+    for (const auto rel_point : shape.footprint.points) {
+      const double length_to_point = std::hypot(rel_point.x, rel_point.y);
+      if (min_length_to_point > length_to_point) {
+        min_length_to_point = length_to_point;
+      }
+    }
+  }
+
+  throw std::logic_error("The shape type is not supported in obstacle_cruise_planner.");
```

Comment:
@kosuke55 Is this function missing return value ? is this function really used ?

Reply:
oh, thanks!

I check this function is used with `BOUNDING_BOX`
```
  if (shape.type == Shape::BOUNDING_BOX) {
    return std::min(shape.dimensions.x / 2.0, shape.dimensions.y / 2.0);
   }
```